packages/protocol/test-sol/stability/Reserve.t.sol

```diff
@@ -3,15 +3,17 @@ pragma solidity ^0.5.13;
 pragma experimental ABIEncoderV2;
 
 import "openzeppelin-solidity/contracts/math/SafeMath.sol";
-import "celo-foundry/Test.sol";
+import { Test, console2 as console } from "celo-foundry/Test.sol";
```

Comment:
why console2?

Reply:
this is for testing, it'll be removed later